interchaintest/helpers/cosmwasm.go

```diff
@@ -38,6 +43,48 @@ func SetupContract(t *testing.T, ctx context.Context, chain *cosmos.CosmosChain,
 	return codeId, contractAddr
 }
 
+func SetupContractWithAdmin(t *testing.T, ctx context.Context, chain *cosmos.CosmosChain, creatorKeyName string, adminAddress string, fileLoc string, message string) (codeId, contract string) {
```

Comment:
I am going to replace this with SetupContract with extraFlags since this is similar. Then you can just --admin in the SetupContract(...) func

Reply:
resolved with 782ff97a1a238b9aae03a1363dd37676e724a04e